tsconfig.json

```
@@ -2,6 +2,7 @@
   "compilerOptions": {
     "target": "ES2020",
     "module": "ES2020",
+    "lib": ["ES2020", "ES2024.Promise"],
```

```suggestion
    "lib": ["ES2020", "ES2024.Promise", "DOM", "DOM.AsyncIterable", "DOM.Iterable", "ScriptHost", "WebWorker.ImportScripts"],
```
You have to re-add all of the default libs because this overwrites the whole thing rather than being additive.